src/core/utils/props.ts

```diff
@@ -9,6 +9,13 @@ export type ClassNameProps = {
   className?: string;
 };
 
+export type StylingProps = {
+  /**
+   * Custom CSS style properties.
+   */
+  style?: React.CSSProperties;
+} & ClassNameProps;
+
 export type CommonProps = {
```

Comment:
Now CommonProps should take StylingProps, I guess...

Reply:
...

Reply:
Can you also merge main? Then I think we can complete this.